CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
`/src/system/libs/swig]>VERBOSE=1 make`

Building CXX object src/system/libs/swig/CMakeFiles/_Config.dir/ConfigPYTHON_wrap.cxx.o
cd /Users/gilles/Development/seiscomp3-git/build-seiscomp3-osx-jakarta-2016/src/system/libs/swig && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++   -DSC_ALL_SHARED -D_Config_EXPORTS -D_REENTRANT -I/Users/gilles/Development/seiscomp3-git/seiscomp3-src-osx-jakarta-2016/src/system/libs -I/Users/gilles/Development/seiscomp3-git/build-seiscomp3-osx-jakarta-2016/src/system/libs -I/Users/gilles/Development/seiscomp3-git/seiscomp3-src-osx-jakarta-2016/src/system/libs/swig  -fPIC   -DSHARED_LIBRARY_SUFFIX="\".dylib\"" -DSHARED_MODULE_SUFFIX="\".so\"" -o CMakeFiles/_Config.dir/ConfigPYTHON_wrap.cxx.o -c /Users/gilles/Development/seiscomp3-git/seiscomp3-src-osx-jakarta-2016/src/system/libs/swig/ConfigPYTHON_wrap.cxx
/Users/gilles/Development/seiscomp3-git/seiscomp3-src-osx-jakarta-2016/src/system/libs/swig/ConfigPYTHON_wrap.cxx:155:11: fatal error: 'Python.h'
      file not found
#include <Python.h>
          ^
1 error generated.
make[2]: *** [src/system/libs/swig/CMakeFiles/_Config.dir/ConfigPYTHON_wrap.cxx.o] Error 1
make[1]: *** [src/system/libs/swig/CMakeFiles/_Config.dir/all] Error 2
make: *** [all] Error 2
[gilles@sangoku /Users/gilles/Development/seiscomp3-git/build-seiscomp3-osx-jakarta-2016/src/system/libs/swig]>